GTG/gtk/browser/browser.py

```diff
@@ -487,19 +471,19 @@ def restore_state_from_conf(self):
         self.builder.get_object("vpaned1").connect('notify::position',
                                                    on_bottom_pan_position)
 
-        toolbar = self.config.get("toolbar")
-        if toolbar:
-            self.builder.get_object("view_toolbar").set_active(1)
-        else:
-            self.toolbar.hide()
-            self.builder.get_object("view_toolbar").set_active(False)
+        # toolbar = self.config.get("toolbar")
+        # if toolbar:
+        #     self.builder.get_object("view_toolbar").set_active(1)
+        # else:
+        #     self.toolbar.hide()
+        #     self.builder.get_object("view_toolbar").set_active(False)
 
-        quickadd_pane = self.config.get("quick_add")
-        if quickadd_pane:
-            self.builder.get_object("view_quickadd").set_active(True)
-        else:
-            self.quickadd_pane.hide()
-            self.builder.get_object("view_quickadd").set_active(False)
+        # quickadd_pane = self.config.get("quick_add")
+        # if quickadd_pane:
+        #     self.builder.get_object("view_quickadd").set_active(True)
+        # else:
+        #     self.quickadd_pane.hide()
+        #     self.builder.get_object("view_quickadd").set_active(False)
```

Comment:
I think we can get rid of both blocks. 

Also, would you mind to merge back upstream changes in your branch? I think there might be a merge conflict. 

After you do that, please add todo to file GTG/core/config.py to remove toolbar and quick_add options from configuration (as we probably won't need them by release time)